code/arith.py

```diff
@@ -6,28 +7,35 @@ def main():
 
     operator = sys.argv[1]
     assert operator in ['add', 'subtract', 'multiply', 'divide'], (
-        'Operator is not one of add, subtract, multiply, or divide: '
-        'bailing out')
+        '''Operator is not one of add, subtract, multiply, or divide:
+        bailing out''')
     try:
         operand1, operand2 = float(sys.argv[2]), float(sys.argv[3])
     except ValueError:
         print('Cannot convert input to a number: bailing out')
-        return
+        return None
```

Comment:
I think using `return None` instead of `return` could be a helpful change to point out what is happening to learners.

Reply:
Probably, I can't recall if there is any mention of the difference between `return`, `return None`, and an empty string in the lessons. Again, it's a matter of consistence rather than my personal preference. Should I adjust all of the functions?